compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -1471,6 +1474,9 @@ public void visit(BLangTypeDefinition typeDefinition) {
         boolean hasTypeInclusions = false;
         NodeKind typeNodeKind = typeDefinition.typeNode.getKind();
         if (typeNodeKind == NodeKind.OBJECT_TYPE || typeNodeKind == NodeKind.RECORD_TYPE) {
+            if (definedType.tsymbol.scope == null) {
+                definedType.tsymbol.scope = new Scope(definedType.tsymbol);
+            }
```

Comment:
What's this change about? 🤔

Reply:
While we define symbols using tsymbols for objects/rec, a scope was created at the time. This is to make sure scope is created